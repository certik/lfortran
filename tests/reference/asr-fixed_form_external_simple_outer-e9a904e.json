{
    "basename": "asr-fixed_form_external_simple_outer-e9a904e",
    "cmd": "lfortran --fixed-form --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/fixed_form_external_simple_outer.f",
    "infile_hash": "dda4e75979612f13dedcc055a7dbb0e28f507472e4e168293acc4e3f",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-fixed_form_external_simple_outer-e9a904e.stdout",
    "stdout_hash": "3ccbbcceceb94ff40c11344ff9069687cc882f4a57f2d57cb1e47d04",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}