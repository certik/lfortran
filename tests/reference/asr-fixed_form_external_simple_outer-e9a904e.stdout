(TranslationUnit (SymbolTable 1 {g: (Function (SymbolTable 4 {a: (Variable 4 a Unspecified () () Default (Integer 4 []) Source Public Required .false.)}) g [(Var 4 a)] [] [(Print () [(Var 4 a)] () ())] () Source Public Implementation () .false. .false. .false.), main: (Program (SymbolTable 2 {a: (Variable 2 a Local () () Default (Integer 4 []) Source Public Required .false.)}) main [] [(= (Var 2 a) (IntegerConstant 1 (Integer 4 [])) ()) (SubroutineCall 1 g () [((Var 2 a))] ())])}) [])
